### PR TITLE
DEVDOCS-5708: [new] Customers V3, GET/ POST/ PUT singular endpoints for  metafield customers

### PR DESCRIPTION
**Pull request by bc-tgomez**
# [DEVDOCS-5708]

## What changed?

* 

## Release notes draft

* 

## Anything else?

ping @bc-tayvs 

[DEVDOCS-5708]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-5708?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: reference/customers.v3.yml [hunk 1]
```diff
@@ -1621,15 +1623,299 @@ paths:
         name: customerId
         in: path
         required: true
+  '/customers/metafield':
```
Comment (bc-tayvs) [inline reference/customers.v3.yml:1626]: Here should be two path families/patterns 

```/customers/:customerId/metafields```  (GET and POST) 
Get all metafield for customer AND create metafield

```/customers/:customerId/metafields/:metafieldId``` (GET, PUT, DELETE)
Get, Update, and delete metafield by customerId + metafield

Thank you for pointing on routing config. I found that we haven't created getAllMetafieldByCustomerId. Will add it ASAP

File: reference/customers.v3.yml [hunk 2]
```diff
@@ -2672,6 +2818,193 @@ components:
...
+            - key
+            - value
+            - permission_set
+            - resource_type
+            - resource_id
+            - description
+            - id
+            - date_created
+            - date_modified     
+    MetaFieldCollectionResponse:
```
Comment (bc-andreadao) [inline reference/customers.v3.yml:2914]: There seems to be another `MetafieldCollectionResponse` on [Line 4471](https://github.com/bigcommerce/docs/blob/5f54ab0b4d37da43eb707af515ed5c5dc10b77ba/reference/customers.v3.yml#L4471). If they are different, you might have to rename yours

File: reference/customers.v3.yml [hunk 3]
```diff
@@ -2672,6 +2818,193 @@ components:
...
+      description: Response payload for the BigCommerce API.
+      content:
+        application/json:
+          schema:
+            type: object
+            properties:
+              data:
+                type: array
```
Comment (bc-andreadao) [inline reference/customers.v3.yml:2922]: I think there is a mistake with how `data` is documented here (data is an array, but there's no information about it)

Comment (bc-tgomez) [reply reference/customers.v3.yml:2922]: It mirrors the same response as line 2821

**Pull request merged**